docs/volunteering/git-and-github.md

```diff
@@ -0,0 +1,47 @@
+# Git & GitHub
+
+We use [git][git] and [GitHub][github] for a lot of our collaborative work.
+Git is a version control system which allows contributors to view a full history
+of our work (whether it's source code for software, designs for our kit hardware
+or something else) as well as for their contributions to add to that history.
+GitHub provides us with an online platform which is openly readable by everyone
+as well as allowing for review of contributions to share knowledge and maintain
+quality.
+
+A full user guide to either tool is well beyond the scope of the runbook, though
```

Comment:
It might be worth referencing a couple useful resources here to get people started.

https://guides.github.com/ contains a number of really useful guides for things, and https://github.github.com/training-kit/downloads/github-git-cheat-sheet.pdf is just generally a useful cheat sheet.

Reply:
5a037f5